smartmeter_datacollector/smartmeter/hdlc_dlms_parser.py

```diff
@@ -21,17 +21,20 @@
 
 
 class HdlcDlmsParser:
-    HDLC_BUFFER_MAX_SIZE = 10000
+    HDLC_BUFFER_MAX_SIZE = 5000
```

Comment:
Is there any reason for reducing this?

Reply:
Yes, I wanted to have the capacity of about 10 times the default L+G meter message, but first missed that 10000 hex characters are indeed only 5000 bytes...
In a separate commit now.